src/pages/documentation/troubleshooting-guides/tabs/push-notifications.md

```diff
@@ -61,7 +61,7 @@ If SDK privacy status is `optedout`, the push identifier will not be set.
 
    ![ECID network request for push token sync](../assets/push-notifications/push-identifier.png)
 
-<Variant platform="ios" task="receive" repeat="5"/>
+<Variant platform="ios" task="receive" repeat="4"/>
```

Comment:
Missed this before! 
1. Preserve the indentation on L76 and you do not need to modify the repeat count. (Recommend this as this is in-line with formatting on the rest of the page)
2. If you remove the indentation then repeat count will have to be increased to 6 (and not decreased to 4). Currently, there is an issue with the content in iOS/ Android tabs because of removal of indentation and decrease of repeat count.

Reply:
@calebk1 can you take this one?

Reply:
I think we I tested the repeat before without the removal of indents, it wasn't working - which is why I removed it in this PR (because there was SOMETHING weird going on).

Reply:
OK, it's all working now so ???